test/quizzes_test.py

```diff
@@ -1,20 +1,63 @@
 import unittest
-
 from app.controllers.quizzes_controller import QuizzesController
+from datetime import datetime
 
 class QuizzesTest(unittest.TestCase):
 
     def setUp(self):
         # Run tests on non-production data
-        self.ctrl = QuizzesController('quizzes_test.py')
-        
+        self.ctrl = QuizzesController('quizzes_test.json')
+
     def test_expose_failure_01(self):
         """
-        Implement this function and two more that
-        execute the code and make it fail.
+        This test case aims to expose a crash when adding a quiz with missing parameters.
+        Expected crash: quizzes_controller.py, line 64
         """
-        self.assertTrue(True, 'Example assertion.')
+
+        self.ctrl.clear_data()
+        # Not passing the available_date param
+        # This will give missing positional argument: 'available_date'
```

Comment:
This seems correct. Maybe, we can also try to change the variable type instead of passing a missing argument.

Reply:
Notes, I have changes variable type of title.